### ISSUE TITLE
Research Request - Transit Agency Peer Groups subtask: NTD Characteristics

### ISSUE THREAD

**Issue by KatrinaMKaiser** [cal-itp/data-analyses#1442]
Complete the below when receiving a research request, and continue to add to this issue as you receive additional details and produce deliverables. Be sure to also add the appropriate project-level label to this issue (eg gtfs-rt, DLA).

## Research Question
**Single sentence** description:  How do we classify CA transit agencies into peer groups based on their characteristics in NTD data?
  
**Detailed** description:

## How will this research be used?
subtask of #1415 

## Stakeholders & End-Users
Who is asking the question, and who is the audience?

## Metrics
* correlation matrices
* Exploratory principal components, k-means clustering or similar

## Data sources
* **Cal-ITP data sources**:
    * NTD Data 

* **External data sources**:

* **Remaining data source questions**:

## Deliverables
Exploratory notebook with statistically determined groupings based on relevant NTD variables
Clean agency-level table with relevant variables that can be integrated into bigger multivariate analysis

## Timeline of deliverables
Estimated completion date TBD

**Comment by csuyat-dot** [comment]
## Draft Workplan

### Possible Data Sources
NTD
- something granular with multiple years of data, 
- something with main NTD metrics like UPT, VRM, VRH, VOMS, Mode, TOS, 

`cal-itp-data-infra.mart_ntd_annual_reporting.fct_metrics`
Reports years 2022-2023.
column include: 
- Agency
- Agency VOMS
- cost per hour
- cost per passenger
- cost per passenger mile
- fare revenue earned
- fare revenue per total (?)
- fare revenue per unlinked (?)
- mode
- mode VOMS
- ntd id
- org type
- passenger miles
- passengers per hour
- report year
- report type
- state
- total operating expense
- type of service
- unlinked passenger trips
- vehicle revenue hour
- vehicle revenue miles
- CT district

`mart_ntd_funding_and_expenses.fct_service_data_and_operating_expenses_time_series_by_mode_upt...voms...vrm...vrh`
Report years 1991-2023. BUT need may need to merge tables together to get all the variables into one 
columns include:
- agency name
- city
- mode
- ntd id
- reporter type
- reporting module
- service
- state
- CT district
- vrm
- vrh
- upt
- voms

### scikit learns and clustering 
https://scikit-learn.org/stable/modules/clustering.html

can `from sklearn.cluster import KMeans from sklearn.preprocessing import StandardScaler` in a notebook.
would need to encode categorical variables for kmeans to work. 

###

**Comment by csuyat-dot** [comment]
Found research article. [Comparing Transit Agency Peer Groups using Cluster Analysis](https://journals.sagepub.com/doi/10.1177/0361198119852074)

**Comment by raebot** [comment]
@csuyat-dot would you be willing to share a PDF of the article [Comparing Transit Agency Peer Groups using Cluster Analysis](https://journals.sagepub.com/doi/10.1177/0361198119852074) here or are there login credentials we can use? It's behind a paywall. Thanks!

**Comment by csuyat-dot** [comment]
[ederer-et-al-2019-comparing-transit-agency-peer-groups-using-cluster-analysis (1).pdf](https://github.com/user-attachments/files/20957216/ederer-et-al-2019-comparing-transit-agency-peer-groups-using-cluster-analysis.1.pdf)
